src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -999,58 +998,53 @@ public int GetCookiesCount()
             int count = 0;
             lock (SyncRoot)
             {
-                foreach (CookieCollection cc in _list.Values)
+                foreach (KeyValuePair<string, CookieCollection> pair in _list)
                 {
+                    CookieCollection cc = pair.Value;
                     count += cc.Count;
                 }
             }
             return count;
         }
 
-        public ICollection Values
```

Comment:
> Hence my questioning whether this was dead code (after your changes). I take it the answer is "yes" ;)

Ah, yeah, the answer is "yes", sorry! :) From my perspective I deleted `Values` first then fixed the call sites, so I wasn't thinking of it as "dead code" because it was already gone. I thought you were asking if it was dead code before my changes.